internals-js/src/definitions.ts

```diff
@@ -1397,7 +1401,7 @@ export class Schema {
     }
 
     if (errors.length > 0) {
-      throw ErrGraphQLValidationFailed(errors);
+      throw ErrGraphQLValidationFailed([...errors]);
```

Comment:
This is ok, but probably also fine to just case `as GraphQLError[]` here. The underlying `err` from the `core-schema` package should probably just accept readonly or mutable instead.